index.html

```diff
@@ -61,6 +61,172 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
+
+      <div class="project-btns">
+        <button class="project-btns-list">css</button>
+        <button class="project-btns-list">html</button>
+        <button class="project-btns-list" id="bts">bootstrap</button>
+        <button class="project-btns-list">Ruby</button>
+      </div>
+      <div class="project-bttn">
+        <button class="project-btn">See Project</button>
+      </div>
+      </div>
+          </div>
+        </div>
+        
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
+        <button class="project1-btn">See project</button>
+      </div>
+      <div class="projects-primary">
+        <div class="projects-heading1">
+          <h3>Professional art <span>printing Data</span></p>
+        </div>
+        <div class="projects-disc1">
+          <p>A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="project1-bttns">
+          <button class="project1-btns">html</button>
+          <button class="project1-btns">bootstrap</button>
+          <button class="project1-btns">Ruby</button>
+        </div>
```

Comment:
- [ ] I am not sure if you actually have a plan to make this part of your code interactive with users but going by Figma design, it is just a list of the skills and stacks you used on a specific project. As a convention and best practice, it's best to use a `ul` and `li` to handle this section. Kindly list your skills with an `unordered list` :pray:. In fact, it will make your styling very easy :wink: